dashboard/app/assets/javascripts/utils.js.erb

```diff
@@ -9,18 +9,6 @@ var addClickTouchEvent = function(element, handler) {
   });
 };
 
-function onURLBlocked(base_url, callback) {
-  $('<img width=1 height=1>')
-      .on("error", callback)
-      .attr("src", base_url + "?" + Math.random())
```

Comment:
Not sure if there's other logic that supplants this, it looks like the `"?" + Math.random()` is no longer in the new version.

That was included so if e.g. someone visits the website on their laptop in an unblocked environment (home), then visits it in a blocked-environment (school), they'd still have the blocking identified. In tests, loading a cached image resulted in not hitting the error callback (successful load)